app/controllers/api/v2/providers_controller.rb

```diff
@@ -33,6 +33,14 @@ def show
         render jsonapi: @provider, include: params[:include]
       end
 
+      # GET /providers/:code/show_from_accredited_body
+      # Allows accredited bodies to see a provider they accredit
```

Comment:
```suggestion
      # Allows an accredited body to see a training provider they accredit
```

Reply:
I have now changed to use your suggestion